tdrs-backend/tdpservice/reports/views.py

```diff
@@ -17,15 +18,43 @@
 logger = logging.getLogger()
 
 
+class ReportFileFilter(filters.FilterSet):
+    """Filters that can be applied to GET requests as query parameters."""
+
+    # Override the generated definition for the STT field so we can require it.
+    stt = filters.NumberFilter(field_name='stt_id', required=True)
+
+    class Meta:
+        """Class metadata linking to the ReportFile and fields accepted."""
+
+        model = ReportFile
+        fields = ['stt', 'quarter', 'year']
```

Comment:
This FilterSet allows query parameters to be supplied to the list endpoint which allows the frontend to retrieve only the reports needed for the current selections. Example API call:
```
GET /v1/reports?stt=10&quarter=Q1&year=2021
```